src/redux/stats/stats.reducer.js

```diff
@@ -15,7 +15,7 @@ import {
 
 const { labels, doughnut, bar } = config;
 
-const initialState = {
+export const initialState = {
```

Comment:
Why you export this value if you don't use it?

Reply:
i got it,i removed it